packages/code-du-travail-frontend/src/layout/css/Headings.js

```diff
@@ -18,15 +18,31 @@ export default createGlobalStyle`
     font-weight: normal;
     font-size: ${fonts.sizes.headings.large};
     font-family: 'Merriweather', serif;
+    @media (max-width: ${breakpoints.mobile}) {
```

Comment:
Since we have heading, shouldn't we remove that globalStyles ( or only add it in fichesMT? ) and add adpative styles to headings to ?

Reply:
The main thing is that heading components have stripes (altough we could have add option to render them without stripes) but then why should we use a component ? Plus as you say they still are raw sources of content. From fiches MT to contributions, the amount of work to get to this right would probably require several pull requests. Maybe when we'll have more time